components/InputReleaseType/InputReleaseType.jsx

```diff
@@ -11,15 +11,21 @@ const releaseTypes = [
   { id: 9, text: "Demo" },
 ]
 
-export default function InputReleaseType({ type, onChange }) {
+export default function InputReleaseType({ type, dispatch }) {
```

Comment:
You can leave this as `onChange` so that we know that the `dispatch` is being called in a change event.